presto-main/src/main/java/com/facebook/presto/metadata/MetadataManager.java

```diff
@@ -901,6 +912,13 @@ private ConnectorMetadata getMetadataForWrite(Session session, ConnectorId conne
         return new JsonCodecFactory(provider).jsonCodec(ViewDefinition.class);
     }
 
+    private Predicate<TableLayoutResult> layoutHasAllDesiredColumn(Optional<Set<ColumnHandle>> desiredColumns)
```

Comment:
How about renaming `layoutHasAllDesiredColumn` as `hasDesiredColumns` (plural and it's clear from the context that we are working on layouts)?